### DISCUSSION TITLE
New Client Generator: Light and Dark logo options.

### DISCUSSION THREAD

**Discussion by StratISaurus** [rustdesk/rustdesk-server-pro#218] [Ideas]
As the RustDesk client follows the Windows color mode (dark or light). It would be nice to be able to define a light and dark logo in the custom client generator. Our logo looks good on eye searing light mode, but has low contrast on dark mode. Thank you.

**Comment by rustdesk** [comment]
you'd better design a logo with some background, that is what we all do with our logo on github readme.

we may allow theme later, but it wont come quickly.

**Comment by rallisf1** [comment]
I'd like this as well. It should be easy to implement. Just duplicate the logo field and set the client to switch the logo image based on the dark mode value.

**Comment by marhalm** [comment]
Our profile is dark logo on light and light logo on dark so would like this also to be able to have two different logos.

**Comment by neurosys-zero** [comment]
+1

**Comment by rustdesk** [comment]
@fufesou

**Comment by st0rm-cr0w** [comment]
Just tested it on custom clients and it works flawlessly. Thank you for your work, guys!
